R/coops.R

```diff
@@ -0,0 +1,28 @@
+#' Get NOAA co-ops data
+#'
+#' @name coops
+#' @param station_name numeric
+#' @param begin_date numeric date in yyyymmdd format
+#' @param end_date numeric date in yyyymmdd format
+#' @param datum character choice of MHHW, MHW, MTL, MSL, MLW, MLLW, NAVD, STND
+#' @param product character choice of water_level, air_temperature, water_temperature, wind, air_pressure, air_gap, conductivity, visibility, humidity, salinity, hourly_height, high_low, daily_mean, monthly_mean, one_minute_water_level, predictions, datums, currents.
```

Comment:
@jsta please shorten lines to less than 100 so easier to read, and check doesn't complain